docker-compose.yml

```diff
@@ -0,0 +1,9 @@
+bot:
+  build: .
+  volumes:
+    - .:/web
+  ports:
+    - "8800:80"
+  command: /bin/bash -c "cd /web/;
```

Comment:
Requirements can change between your two docker-compose up. 
Rebuild image every time is waste of time.